dll/win32/shlwapi/appcompat.c

```diff
@@ -0,0 +1,443 @@
+/*
+ * PROJECT:     ReactOS Shell
+ * LICENSE:     LGPL-2.0-or-later (https://spdx.org/licenses/LGPL-2.0-or-later)
```

Comment:
Do we want LGPL or MIT for our new code, here in a DLL? (cc @learn-more @DarkFire01)

Reply:
My preference is always MIT, but this module might already be infected...